system-test/data/read-rows-retry-test.json

```diff
@@ -151,8 +171,8 @@
         "keys": ["a", "b", "x"]
       },
       "request_options": [
-        { "rowKeys": [ "a", "b", "x" ] },
-        { "rowKeys": [ "x" ], "rowRanges": [ { "startKeyOpen": "c" } ] }
+        { "rowKeys": [ "a", "b", "x" ], "rowRanges": [] },
```

Comment:
I think this test was wrong, it added a new range for a request that only had row keys right?

Reply:
It does seem strange that we would actually add a row range on the second request. What happens if we remove `"rowRanges": []` though?